### DISCUSSION TITLE
Request feature for integration with Auth0 or 3rd party auth process

### DISCUSSION THREAD

**Discussion by hiccuphaddock** [spencerwooo/onedrive-vercel-index#282] [Ideas]
- Current: protection only can be proceed by set the `protectedRoutes`, which only limited the special folder content.
- Feature request: login authorization can be controlled by user base roles, which can be done by integrating with Auth0 or other public auth platform.
- Reference: https://auth0.com/docs/quickstart/webapp/nextjs/01-login

Would you consider the integration with Auth0? It will be nice for better RBAC function.
appreciate for your nice work~

**Comment by spencerwooo** [comment]
maybe, in the future.
